lib/pact_broker/client/cli/matrix_commands.rb

```diff
@@ -48,6 +48,7 @@ def can_i_deploy(*ignored_but_necessary)
             long_desc "Checks if the specified pacticipant version is compatible with the configured main branch of each of the pacticipants with which it is integrated."
             method_option :pacticipant, required: true, aliases: "-a", desc: "The pacticipant name. Use once for each pacticipant being checked."
             method_option :version, required: false, aliases: "-e", desc: "The pacticipant version. Must be entered after the --pacticipant that it relates to."
+            method_option :ignore, required: false, banner: "PACTICIPANT", desc: "The pacticipant name to ignore. Use once for each pacticipant being ignored. A specific version can be ignored by also specifying a --version after the pacticipant name option. The environment variable PACT_BROKER_CAN_I_MERGE_IGNORE may also be used to specify a pacticipant name to ignore, with commas to separate multiple pacticipant names if necessary."
```

Comment:
Can you specify both a pacticipant and a version to ignore via the environment variable? e.g. `PACT_BROKER_CAN_I_MERGE_IGNORE="Foo=1.2.4,Bar=4.5.6"` type thing? (I assume not)